src/main/kotlin/translations/sorting/TranslationTemplateConfigurable.kt

```diff
@@ -61,6 +75,33 @@ class TranslationTemplateConfigurable(private val project: Project) : Configurab
         }
 
     private fun init() {
+        dialogPanel = panel {
+            val translationSettings = TranslationSettings.getInstance(project)
+            row {
+                checkBox(MCDevBundle("minecraft.settings.translation.force_json_translation_file"))
+                    .bindSelected(translationSettings::isForceJsonTranslationFile)
+            }
+
+            lateinit var allowConvertToTranslationTemplate: ComponentPredicate
+            row {
+                val checkBox = checkBox(MCDevBundle("minecraft.settings.translation.use_custom_convert_template"))
+                    .bindSelected(translationSettings::isUseCustomConvertToTranslationTemplate)
+                allowConvertToTranslationTemplate = checkBox.selected
+            }
+
+            row {
+                textField().bindText(translationSettings::convertToTranslationTemplate)
+                    .enabledIf(allowConvertToTranslationTemplate)
+            }
+
+            onApply {
+                for (project in ProjectManager.getInstance().openProjects) {
+                    ProjectView.getInstance(project).refresh()
+                }
```

Comment:
Why is this needed?

Reply:
[9c4a0c0](https://github.com/minecraft-dev/MinecraftDev/pull/2292/commits/9c4a0c09a8260f79ed2a4c74ed5ff73ad0fc9d24)